src/nvim/eval.lua

```diff
@@ -291,141 +291,141 @@ return {
     rpcstart={args={1, 2}},
     rpcstop={args=1},
     rubyeval={args=1, base=1},
-    screenattr={args=2},
-    screenchar={args=2},
-    screenchars={args=2},
+    screenattr={args=2, base=1},
+    screenchar={args=2, base=1},
+    screenchars={args=2, base=1},
     screencol={},
-    screenpos={args=3},
+    screenpos={args=3, base=1},
     screenrow={},
-    screenstring={args=2},
-    search={args={1, 4}},
-    searchcount={args={0,1}},
-    searchdecl={args={1, 3}},
+    screenstring={args=2, base=1},
+    search={args={1, 4}, base=1},
+    searchcount={args={0, 1}, base=1},
+    searchdecl={args={1, 3}, base=1},
     searchpair={args={3, 7}},
     searchpairpos={args={3, 7}},
-    searchpos={args={1, 4}},
+    searchpos={args={1, 4}, base=1},
     serverlist={},
     serverstart={args={0, 1}},
     serverstop={args=1},
-    setbufline={args=3},
-    setbufvar={args=3},
-    setcharsearch={args=1},
-    setcmdpos={args=1},
-    setenv={args=2},
+    setbufline={args=3, base=3},
+    setbufvar={args=3, base=3},
+    setcharsearch={args=1, base=1},
+    setcmdpos={args=1, base=1},
+    setenv={args=2, base=2},
     setfperm={args=2, base=1},
-    setline={args=2},
-    setloclist={args={2, 4}},
-    setmatches={args={1, 2}},
-    setpos={args=2},
-    setqflist={args={1, 3}},
-    setreg={args={2, 3}},
-    settabvar={args=3},
-    settabwinvar={args=4},
-    settagstack={args={2, 3}},
-    setwinvar={args=3},
-    sha256={args=1},
-    shellescape={args={1, 2}},
-    shiftwidth={args={0, 1}},
-    sign_define={args={1, 2}},
-    sign_getdefined={args={0, 1}},
-    sign_getplaced={args={0, 2}},
-    sign_jump={args={3, 3}},
-    sign_place={args={4, 5}},
-    sign_placelist={args={1}},
-    sign_undefine={args={0, 1}},
-    sign_unplace={args={1, 2}},
-    sign_unplacelist={args={1}},
-    simplify={args=1},
+    setline={args=2, base=2},
+    setloclist={args={2, 4}, base=2},
+    setmatches={args={1, 2}, base=1},
+    setpos={args=2, base=2},
+    setqflist={args={1, 3}, base=1},
+    setreg={args={2, 3}, base=2},
+    settabvar={args=3, base=3},
+    settabwinvar={args=4, base=4},
+    settagstack={args={2, 3}, base=2},
+    setwinvar={args=3, base=3},
+    sha256={args=1, base=1},
+    shellescape={args={1, 2}, base=1},
+    shiftwidth={args={0, 1}, base=1},
+    sign_define={args={1, 2}, base=1},
+    sign_getdefined={args={0, 1}, base=1},
+    sign_getplaced={args={0, 2}, base=1},
+    sign_jump={args=3, base=1},
+    sign_place={args={4, 5}, base=1},
+    sign_placelist={args=1, base=1},
+    sign_undefine={args={0, 1}, base=1},
+    sign_unplace={args={1, 2}, base=1},
+    sign_unplacelist={args=1, base=1},
```

Comment:
Wonder if some ports made the same mistake. Didn't expect `{1}` to expand to `{1, max}`.